config/mycnf/default.cnf

```diff
@@ -1,35 +1,35 @@
-# basic config parameters for all db instances in the grid
+# Global configuration that is auto-included for all MySQL/MariaDB versions
 
-sql_mode = STRICT_TRANS_TABLES
-binlog_format = statement
-character_set_server = utf8
-collation_server = utf8_general_ci
-connect_timeout = 30
 datadir = {{.DataDir}}
-expire_logs_days = 3
-innodb_buffer_pool_size = 32M
 innodb_data_home_dir = {{.InnodbDataHomeDir}}
-innodb_flush_log_at_trx_commit = 2
-innodb_lock_wait_timeout = 20
 innodb_log_group_home_dir = {{.InnodbLogGroupHomeDir}}
 log-error = {{.ErrorLogPath}}
-long_query_time = 2
-max_allowed_packet = 64M
-max_connections = 500
```

Comment:
I see we've dropped the use of max_connections from the bundled config files completely.  Is this a concern?  The defaults are rather low....

Reply:
It might be a problem; the error messages are not always clear what pools will need to be changed. Thanks for the feedback, I'll bump it back to 500 for now, and we can figure out if there is a better long term fix after documenting https://github.com/vitessio/website/issues/302